nixos/modules/tasks/cpu-freq.nix

```diff
@@ -4,22 +4,43 @@ with lib;
 
 let
   cpupower = config.boot.kernelPackages.cpupower;
-  cfg = config.powerManagement;
+  cfg = config.powerManagement.cpufreq;
 in
 
 {
   ###### interface
 
   options = {
 
-    powerManagement.cpuFreqGovernor = mkOption {
+    powerManagement.cpufreq.governor = mkOption {
       type = types.nullOr types.str;
       default = null;
       example = "ondemand";
       description = ''
         Configure the governor used to regulate the frequence of the
         available CPUs. By default, the kernel configures the
-        performance governor.
+        performance governor, although this may be overwriten in your
+        hardware.nix file.
+
+        Often used values: "ondemand", "powersave", "performance"
+      '';
+    };
+
+    powerManagement.cpufreq.max = mkOption {
+      type = types.nullOr types.ints.unsigned;
+      default = null;
+      example = 2200000;
+      description = ''
+        The maximum frequency the CPU will use.  Defaults to the maximum possible.
+      '';
+    };
+
+    powerManagement.cpufreq.min = mkOption {
```

Comment:
One minor suggestion:

```nix
{
  options.powerManagement.cpufreq = {
    governor = mkOption ...;
    min = mkOption ...;
    max = mkOption ...;
  };
  # ...
}
```

Reply:
Another good suggestion.  This change has been made in commit 383fbfb.

I think this PR should be ready for a final review (and hopefully merging!).